### ISSUE TITLE
FHD Setting in manifest file messes things up

### ISSUE THREAD

**Issue by jsonnentag** [rrirower/DP2SceneGraphWizard#72]
I decided to look at what the updated channels look like on another TV that is a 4K one.  Apparently the ONLY setting we should use in the "wizard" is the default HD one, because I originally (and what was approved by Roku just today) put check marks in all 3 resolution options (SD, HD, FHD).  You can see what that looks like on a 4K TV.

I don't know what the resolution is for if not to indicate what the graphics (JPGs) are made for.  (???)  Anyway, the 2nd JPG is what it look like when only HD is in the manifest file.  So I guess I will be sending Roku updated versions, because the one with FHD in the manifest looks terrible.  When checking things before I was looking at a Roku 3 on a 1080p TV and it looked OK on that with FHD in the manifest.  (???)

![FHD_Not_Working](https://github.com/rrirower/DP2SceneGraphWizard/assets/36777806/b0f4a93d-92b6-47af-b000-dbba4a1c5a5a)

![HD_Scaling_But_No_Descriptions](https://github.com/rrirower/DP2SceneGraphWizard/assets/36777806/a49631d0-c388-4c37-a4c1-96dce21c03c7)

**Comment by rrirower** [comment]
See issue #60.  This is a known issue.

**Comment by jsonnentag** [comment]
I guess it isn't known to anybody other that those that already knew about it, because it isn't visible in the Issues list anymore (no #60).  :)

The Wizard just needs to be changed to eliminate that option until it is fixed and actually works.

**Comment by rrirower** [comment]
Fyi... You can filter issues and discussions to show all, open, closed, etc.

**Comment by jsonnentag** [comment]
> Fyi... You can filter issues and discussions to show all, open, closed, etc.

I don't think so.  These are all the options I see and they all refer to sorting by things associated with "you" and I didn't have anything to do with it so why would it appear under any filter listed.  Besides, why would it not be active if it is still a known problem and nothing in the wizard has been changed to at least help prevent it.  (???)

![image](https://github.com/rrirower/DP2SceneGraphWizard/assets/36777806/fba35c30-3fd3-4840-a1c1-8d1b6521781f)

**Comment by rrirower** [comment]
You can click on the 'Open' or 'Closed' text in the header line (first line) to filter only those types of issues.

**Comment by rrirower** [comment]
Can you try something?  Rather than specifying both HD and FHD, just use FHD and see if it works for you.

**Comment by jsonnentag** [comment]
First I tried just changing the manifest of the latest version I sent to Roku.  I changed the "hd" to "fhd" and loaded it onto a 4K Roku TV device.  It still just puts the functional part of the working screen in the top left if it is "fhd" only.  (Everything stretches to fit the screen when set to "hd".)  But I could see how the video thumbnail as a background setting works, because that filled the whole background and only the background JPG was visible in the top left functional part.  So now I can see why I didn't see any change when I turned that option on - I was always using a background.

Anyway, I just ran another round of the wizard just to make sure it wasn't changing or setting something other than in the manifest and with that as "fhd" only it still looks the same - just functional in the top left corner and the rest filled with a stretch to fit background.

**Comment by jsonnentag** [comment]
I just tested it on a Roku 4 attached to a 1080p TV and it has the same result.  Top left functional area only and the rest set to the background color or some kind or dark gray in place of it.  I have the color set to black, but include a JPG for the background.  But it doesn't look as dark of black as the solid setting.  But if the option for the video thumbnail is set it does use that and fill the whole screen with that, while just the top leftmost area is functional as the channel for video selection.

**Comment by rrirower** [comment]
Thanks for the feedback.  Still researching.
